packages/ember-routing/lib/location/none_location.js

```diff
@@ -23,14 +23,30 @@ export default EmberObject.extend({
   path: '',
 
   /**
-    Returns the current path.
+    Will be pre-pended to path.
+
+    @private
+    @property rootURL
+    @default '/'
+  */
+  rootURL: '/',
+
+  /**
+    Returns the current path without `rootURL`.
 
     @private
     @method getURL
     @return {String} path
   */
   getURL() {
-    return get(this, 'path');
+    let path = get(this, 'path');
+    let rootURL = get(this, 'rootURL');
+
+    // remove trailing slashes if they exists
+    rootURL = rootURL.replace(/\/$/, '');
```

Comment:
Can we store the normalized `rootURL` (perhaps in `init`), instead of removing the trailing slash on every `getURL` / `formatURL`?